awx/main/access.py

```diff
@@ -1325,7 +1325,7 @@ class ExecutionEnvironmentAccess(BaseAccess):
 
     def filtered_queryset(self):
         return ExecutionEnvironment.objects.filter(
-            Q(organization__in=Organization.accessible_pk_qs(self.user, 'execution_environment_admin_role')) |
+            Q(organization__in=Organization.accessible_pk_qs(self.user, 'member_role')) |
```

Comment:
Probably should be `'read_role'` now that I've looked at the `Organization` model roles.  Also, I think we need to add `'execution_environment_admin_role` to `Organization.read_role`'s parent roles.

Reply:
`read_role` replaced member role and added to the orgs read_role list